src/System.CommandLine/IArgument.cs

```diff
@@ -3,16 +3,12 @@
 
 namespace System.CommandLine
 {
-    public interface IArgument
+    public interface IArgument : ISymbolBase
```

Comment:
This needs more discussion.

Reply:
I think it is important that we have a "thing that can provide a value" type. I am extremely open on all other issues around it. Although the current design could uptake the remaining entirely independent (by having two BindingSides).

And this conversation can be related to the multiple argument conversation. 

I selected "ISymbolBase" as a way to call out this conversation, and I'm very open on the resolution. Would you like me to open an issue on this specific question as a venue for conversation?

Reply:
Yes, I think this deserves an issue.